src/documents/ComponentConfigurationOptions.md

```diff
@@ -112,40 +110,63 @@ The following options are supported by all component grades:
   </tr>
   <tr>
     <th>See also</th>
-    <td><a href="Invokers.md">Invokers</a></td>
+    <td><a href="InfusionEventSystem.md">Infusion Event System</a></td>
   </tr>
 </table>
 
-### members ###
+### `listeners` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object defining properties to be added to the component object. These can be anything, including methods, strings, objects, etc. Definitions are evaluated as IoC expressions.</td>
+    <td>A record defining listener functions for the events supported by a component.</td>
   </tr>
   <tr>
     <th>Notes</th>
-    <td><code>members</code> differ from <code>invokers</code> in that the arguments of members are not resolved at invocation time.
-        The right-hand-side may contain an <a href="ExpansionOfComponentOptions.md">expander</a> definition, which may perhaps itself resolve onto an <a href="Invokers.md">invoker</a>.</td>
+    <td>Both component developers and integrators can define listeners for events.
+<a href="Invokers.md">Invokers</a> and the `fire` method of other events can be used as listeners here, as well as any function handle resulting from an <a href="ExpansionOfComponentOptions.md">Expanders</a>. 
+Note that as well as being a simple string holding the name of an event on this component, a listener key may also be a full <a href="IoCReferences.md">IoC Reference</a> 
+to any other event held in the component tree (for example <code>"{parentComponent}.events.parentEvent"</code>. As well as being a simple function name, the value associated with the key may be a <a href="InfusionEventSystem.md">Listener Record</a> 
+or else follow the syntax of an invoker indicating that the registered listener receives a different signature from the one that the event has fired (see <a href="EventInjectionAndBoiling.md">Event injection and boiling</a>).</td>
   </tr>
   <tr>
     <th>Example Definition</th>
     <td><pre>
-<code>fluid.defaults("component.name", {
-    members: {
-        member1: "{that}.options.optionsValue",
-        member2: "{theOther}.dom.otherSelector",
+<code>examples.myListener = function (number, condition) {
+    console.log("Event listener received number " + number + " and condition " + condition);
+};
+
+fluid.defaults("examples.eventedComponent", {
```

Comment:
Does it make sense to use "evented" in the name of the component, now that everything is evented by default?